lite/README.md

```diff
@@ -1 +1,141 @@
-# valor-lite: Compute classification, object detection, and segmentation metrics locally.
+# valor-lite: Fast, local machine learning evaluation.
+
+Valor-lite is a lightweight, numpy-based library designed for fast and seamless evaluation of machine learning models. It is optimized for environments where quick, responsive evaluations are essential, whether as part of a larger service or embedded within user-facing tools. By operating without the latency of external systems, Valor-lite ensures an efficient, smooth experience for users.
```

Comment:
> By operating without the latency of external systems, Valor-lite ensures an efficient, smooth experience for users.

suggest deleting this line. different external systems will have different "latencies", so it's not clear which specific system you're talking about here. this line also seems to convey the same information as the line above.